src/main/java/frc/robot/commands/Auto/ZoneShootAuto.java

```diff
@@ -0,0 +1,47 @@
+package frc.robot.commands.Auto;
+
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.robot.TestingDashboard;
+import frc.robot.subsystems.Auto;
+
+import frc.robot.commands.Drive.PIDDriveDistance;
+import frc.robot.commands.Drive.PIDTurnAngle;
+import frc.robot.commands.Shooter.PIDBottomShooter;
+
+public class ZoneShootAuto extends SequentialCommandGroup{
+    public ZoneShootAuto(String zone) {
+        
+            if (zone == "green") {
+                addCommands(
+                new PIDBottomShooter(0.3));
+            }
+            if (zone == "yellow") {
+                addCommands(
+                new PIDBottomShooter(0.4));
+            }
+            if (zone == "blue") {
+                addCommands(
+                new PIDBottomShooter(0.5));
+            }
+            if (zone == "red") {
+                addCommands(
+                new PIDBottomShooter(0.6));
+            }
+    }
+
+    public static void registerWithTestingDashboard() {
+        Auto auto = Auto.getInstance();
+        ZoneShootAuto greenZone = new ZoneShootAuto("green");
+        ZoneShootAuto yellowZone = new ZoneShootAuto("yellow");
+        ZoneShootAuto blueZone = new ZoneShootAuto("blue");
+        ZoneShootAuto redZone = new ZoneShootAuto("red");
+        TestingDashboard.getInstance().registerCommand(auto, "Auto", greenZone);
```

Comment:
We don't need to register each command twice to testing dashboard. Please remove the 4 lines registering with the "Auto" group name and leave the ones registering with the "ZoneShootAuto" group name.